configure

```diff
@@ -4966,6 +4979,38 @@ fi
 define_in_config_if_yes "$_fluidsynth" 'USE_FLUIDSYNTH'
 echo "$_fluidsynth"
 
+#
+# Check for FluidLite
+#
+echocheck "FluidLite"
```

Comment:
What will happen if both fluidsynth and fluidlite are present?

Reply:
In that case, only FluidSynth will be enabled.